app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/agent/check_your_answers.scala.html

```diff
@@ -87,7 +87,7 @@ <h2 class="heading-medium" id="check-your-answer-heading">@messages("agent.check
         @business.businessAddress.map(address => businessAddress(address, address.auditRef))
     </dl>
     <div class="form-group">
-        <a id="remove-business-@businessNumber" href=@uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.agent.routes.RemoveBusinessController.show(business.id).url>@messages("agent.check_your_answers.remove_business", businessNumber)</a>
+        <a id="remove-business-@businessNumber" href="@uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.agent.routes.RemoveBusinessController.show(business.id).url">@messages("agent.check_your_answers.remove_business", businessNumber)</a>@messages("check_your_answers.remove_business", businessNumber)</a>", businessNumber)"</a>
```

Comment:
Not sure what has happened here, looks like a piece of the code was duplicated
```suggestion
        <a id="remove-business-@businessNumber" href="@uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.agent.routes.RemoveBusinessController.show(business.id).url">@messages("agent.check_your_answers.remove_business", businessNumber)</a>
```